### PR TITLE
MAT-8999 Updating tx-qicore-6.0.0.zip package with missing valueSets

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-8999](https://jira.cms.gov/browse/MAT-8999)
(Optional) Related Tickets:

### Summary

- Replace old package with new tx-qicore package that has few of the missing valueSets
- Updated our code to ignore any metadata that MACOS compression adds to the zip

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/madiefhirservice/config/HapiFhirConfig.java [hunk 1]
```diff
@@ -187,20 +188,29 @@ public PrePopulatedValidationSupport buildPrePopulatedValidationSupportFromZip(
 
       ZipEntry entry;
       while ((entry = zipInputStream.getNextEntry()) != null) {
-        if (!entry.isDirectory()) {
+        if (!entry.isDirectory()
+                && !entry.getName().startsWith("__MACOSX")
+                && !entry.getName().endsWith(".DS_Store")) {
```
Comment (adongare) [inline src/main/java/gov/cms/madie/madiefhirservice/config/HapiFhirConfig.java:193]: while its good to have this check, I usually exclude these files while zipping.
`zip -r tx-qicore-6.0.0.zip tx-qicore-6.0.0/ -x "__MACOSX/*" "*.DS_Store" `

**Review by adongare — APPROVED** [review]

File: src/main/java/gov/cms/madie/madiefhirservice/config/HapiFhirConfig.java (hunk 1 above)
Comment (RohitKandimalla) [reply src/main/java/gov/cms/madie/madiefhirservice/config/HapiFhirConfig.java:193]: aah Good to know "-x" . I just added the check as we don't need those files anyways. Folders zipped from Windows are working fine without issues.
